docs/moderator-handbook.md

```diff
@@ -336,33 +337,33 @@ You may come across situations where users seek medical advice or are dealing wi
 
 As a matter of policy, you should avoid talking privately about these matters. Should the situation reflect back to freeCodeCamp, we want to have the conversation(s) on record. Make it clear that we are not medical professionals and that you encourage the user to find professional help.
 
-As difficult as it sometimes can be, avoid giving any tips or advice other than pointing the user in the direction of professional help!
+As difficult as it sometimes can be, avoid giving any tips or advice and rather point the user in the direction of seeking professional help!
 
-If this happens on our chat server: Create a private channel for the user and the mod team. This can be done with the bot's `private` command.
+If this happens on our chat server: Create a private channel for the user and the moderator team. This can be done with the bot's `private` command.
 
-- The user is guaranteed some privacy
-- Public chat is no longer disrupted
-- Other team members can pitch in, should you be uncomfortable dealing with the situation yourself
+- The user is guaranteed some privacy.
+- Public chat is no longer disrupted.
+- Other team members can pitch in, should you feel uncomfortable dealing with the situation yourself.
 
 Helpful URLs:
 
-http://www.suicide.org/international-suicide-hotlines.html
+http://suicide.org/international-suicide-hotlines.html
 
 ## A note on free speech
 
 Sometimes people will defend something offensive or incendiary that they said as "free speech."
 
 This XKCD comic summarizes perfectly most communities' thoughts on free speech. So if someone defends something in the name of "free speech", feel free to send it to them.
 
-<div align="center"><img src='https://aws1.discourse-cdn.com/freecodecamp/original/3X/4/3/43a8b2eafe4c8622e02838f66f1dc6227de32c70.png' width="400" height="400" /></div>
+<div align="center"><img src='https://imgs.xkcd.com/comics/free_speech.png' width="400" height="400" /></div>
```

Comment:
hosting it ourself means we don't ping xkcd servers each time the image is loaded. I wouldn't change this.

Reply:
Yes, I understand, I will definitely need a new image link to update this as the prior link ends up with this:
![image](https://user-images.githubusercontent.com/97711937/181085803-f1f3e9ac-a1b3-4f6f-ba5f-4bc6c1e19fcc.png)

Reply:
images are saved in docs/images for the docs, it can be added there

Reply:
Should I be creating a separate pull request to add an image like that to `freeCodeCamp/docs/images`?

Reply:
as you are changing the link here it should be here

Reply:
Unfortunately I don't see the image of XKCD in `freeCodeCamp/docs/images`.

Reply:
Yeah, it has to be added.

Reply:
So I would have to make a pull request with the file to that subdirectory, correct?

Reply:
You can add it in this PR